src/waveform/widgets/nonglwaveformwidgetabstract.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <QWidget>
+
+#include "waveform/widgets/waveformwidgetabstract.h"
+
+QT_FORWARD_DECLARE_CLASS(QString)
+
+/// NonGLWaveformWidgetAbstract is a WaveformWidgetAbstract & QWidget
```

Comment:
This comment does not add any value. You may explain the design decision or remove the comment at all.

Reply:
This comment is in keeping with the style of the other comment, so I think it's ok to keep it.  What this is lacking is more explanation, for instance what paint method it uses instead